### PR TITLE
370-customer-add-layby-functionality

### PR DESCRIPTION
**Pull request by SibonisoNcube**
Created Purchase File:
1. added layby create
2. added layby get ALL
3. added layby get Specific
4. added layby edit

### CODE REVIEW

File: lib/services/purchases.dart
Comment (bonganimbele) [inline lib/services/purchases.dart:1]: These new methods use a different resource path, thus, I would suggest you put them in their own file

Comment (SibonisoNcube) [reply lib/services/purchases.dart:1]: @bonganimbele please advise.

Comment (bonganimbele) [reply lib/services/purchases.dart:1]: In that case should it not be called layby instead of purchases, since it deals with laybys

**Review by koketsomagagane1219 — APPROVED** [review]
Approved

File: lib/services/purchases.dart
Comment (SibonisoNcube) [reply lib/services/purchases.dart:1]: Okay I will rename it to laybys thank you.

Comment (SibonisoNcube) [reply lib/services/purchases.dart:1]: @bonganimbele I made the changes, please check.

**Review by koketsomagagane1219 — APPROVED** [review]
Approved

**Review by bonganimbele — APPROVED** [review]

**Pull request merged**